src/psyclone/transformations.py

```diff
@@ -543,6 +555,12 @@ def apply(self, node, options=None):
         :param bool options["independent"]: whether to add the "independent" \
                 clause to the directive (not strictly necessary within \
                 PARALLEL regions).
+        :param bool options["sequential"]: whether to add the "seq" clause to \
```

Comment:
No need for line-continuation here.

Reply:
See 1357686886bb0627676797bd1b251ed91508be5a.

Reply:
I'd have let this go but for the need to update the new tests (see below). As it is, you don't seem to have changed this line in the file?

Reply:
Apologies, fixed in 809f01e58e997e9aae68476748bdada7c5d371dd.